pkg/catalog/routes_test.go

```diff
@@ -389,6 +389,38 @@ func TestListAllowedOutboundServices(t *testing.T) {
 	assert.ElementsMatch(actualList, expectedList)
 }
 
+func TestListAllowedOutboundServicesForIdentity(t *testing.T) {
+	assert := assert.New(t)
+	mc := newFakeMeshCatalog()
```

Comment:
I'd remove this and use the mock catalog to be able to make the tests extendable. Using the fake catalog limits the control the test has.